src/main/groovy/io/intrepid/analysis/StaticAnalysis.groovy

```diff
@@ -63,12 +62,8 @@ class StaticAnalysis implements Plugin<Project> {
             reports {
                 xml.enabled = true
                 html.enabled = true
-                xml {
-                    destination "$project.buildDir/reports/pmd/pmd.xml"
-                }
-                html {
-                    destination "$project.buildDir/reports/pmd/pmd.html"
-                }
+                xml.destination = new File("$project.buildDir/reports/pmd/pmdss.xml")
```

Comment:
One question - what does the `ss` at the end of the filename signify?

Reply:
Good catch. That was a typo, I'll remove it